autoscale/src/main/java/com/sequenceiq/periscope/monitor/evaluator/EvaluatorExecutor.java

```diff
@@ -29,7 +29,15 @@
     public final void run() {
         long itemId = getContext().getItemId();
         try {
+            LOGGER.debug("Running EvaluatorExecutor: {}", this);
+            // TODO CB-14972: Have the evaluatorExecutors implement toString with relevant context information.
```

Comment:
We have jiras. I'd much rather leave the TODOs with references in the code - since the code keeps evolving.

Reply:
Removing a bunch of the TODOs from existing code in the next PR. Tracking in CB-15862 instead. Leaving the ones on the new code - since most of them are future enhancements (some get resolved in the error handler patch - to follow)

Reply:
Resolving this conversation, with the removed TODOs ... and rationalized ones on https://github.com/hortonworks/cloudbreak/pull/12183